src/components/article/viewArticleComponents.js

```diff
@@ -171,18 +173,17 @@ ArticleContent.defaultProps = {
   description: '',
   body: '',
 };
-
 RecentArticles.propTypes = {
-  onChange: PropTypes.func,
-  current: PropTypes.number,
-  total: PropTypes.number,
-  articles: PropTypes.array,
+  onChange: PropTypes.string,
+  current: PropTypes.string,
+  total: PropTypes.string,
+  articles: PropTypes.string,
```

Comment:
You should leave these the way they were

Reply:
I think these were changed in the last PR. I dont know why they show as if I am the one that changed them